### PR TITLE
Add creation and deletion of custom metrics to usage doc.

### PR DESCRIPTION
**Pull request by rimey**
I'm soliciting comments on the general outlines of the interface and user experience before submitting a pull request with the implementation.

### CODE REVIEW

File: docs/monitoring-usage.rst [hunk 1]
```diff
@@ -24,8 +24,13 @@ Essential concepts:
 Please refer to the documentation for the `Monitoring API`_ for
 more information.
 
-At present, this client library supports querying of time series,
-metric descriptors, and resource descriptors.
+At present, this client library supports the following features
+of the API:
+
+- Querying of time series.
+- Querying of metric descriptors and resource descriptors.
```
Comment (peaton) [inline docs/monitoring-usage.rst]: *monitored* resource descriptors.
(To match documentation on cloud.google.com.)

**Comment by peaton** [comment]
LGTM

File: docs/monitoring-usage.rst (hunk 1 above)
Comment (rimey) [inline docs/monitoring-usage.rst]: Done.

**Comment by dhermes** [comment]
LGTM. @rimey Do you have merge privileges in this repo?

**Comment by rimey** [comment]
@dhermes The "Merge pull request" button lights up for me, maybe because I'm in GoogleCloudPlatform, but this should be superseded by #1755 anyway. Closing.

@dhermes and @peaton Thank you for the reviews!

**Pull request closed without merging**